include/celero/Print.h

```diff
@@ -41,6 +41,7 @@ namespace celero
 		void Failure(const std::string& x);
 		void Run(std::shared_ptr<Result> x);
 		void Done(std::shared_ptr<Result> x);
+		void DoneExperiment(std::shared_ptr<Experiment> x);
```

Comment:
The problem that I met some difficulties in understanding experiments results in comparing 
celero::timer::ConvertSystemTime(x->getExperiment()->getTotalRunTime()) with x->getRunTime() for each [DONE] line